bundles/org.openhab.binding.openwebnet/README.md

```diff
@@ -368,6 +389,12 @@ sitemap openwebnet label="OpenWebNet Binding Example Sitemap"
     {
           Switch    item=iCENPlusProxyItem  label="My CEN+ scenario" icon="movecontrol"  mappings=[ON="Activate"]
     }
+    
+    Frame label="Alarm activation via AUX command"
+    {
+          Switch item=iAlarm_activation         icon="siren"
```

Comment:
As the channel/item is a string, "Text" looks more appropriate than "Switch" in the sitemap ?

Reply:
I tried to modify as you suggest but in that way the the user is no more able to interact with the sitemap-item, i.e. the sitemap-item become not active!

Reply:
As there are options, Switch or Selection are ok. 
Forget my comment.